crates/scroll/primitives/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "reth-scroll-primitives"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+# alloy
+alloy-primitives.workspace = true
+
+# reth
+reth-codecs.workspace = true
+
+# scroll
+poseidon-bn254 = { git = "https://github.com/scroll-tech/poseidon-bn254", branch = "master", features = ["bn254"] }
```

Comment:
I think we should create a tag or publish this to crates.io instead of using a master branch dependency. I think we should move the dependency to workspace `Cargo.toml` and add a TODO which states that we should change this dep to point to the tag / crate onceds it's published. ref: https://github.com/scroll-tech/poseidon-bn254/issues/6

Reply:
agreed, I left a comment regarding this in the `deny.toml`. Will move to the workspace